validation/Jenkinsfile

```diff
@@ -117,6 +117,10 @@ node {
 
                 sh "./validation/configure.sh"
                 sh "./validation/build.sh"
+                if ("${ENV_VARIABLE}" != "null" && "${ENV_VARIABLE}" != "") {
```

Comment:
can you give this a better name? there are many env vars, what does this one do? you also append the var to the end of the read in text, is that correct (line 122)?